### DISCUSSION TITLE
((EDDI route details)) has not been called

### DISCUSSION THREAD

**Discussion by YenShihHung** [EDCD/EDDI#2673] [EDDI VoiceAttack Q&A] [answered]
Hello I have a problem
No matter I plan a route or jump to other systems ((EDDI route details)), it is not called, and the related variables are Not set.

**Comment by Tkael** [comment]
Hi there. I need a better explanation of how you are invoking RouteDetails and how the program is responding. Would it be possible to share a video? 

Are you using VoiceAttack, the Speech Responder, or the Navigation Monitor? Which specific search are you invoking? Does the search complete or does it time out?

**Reply by YenShihHung** [reply]
Hello, I tried to make a function to manage fuel during the journey in VoiceAttack, but ((EDDI route details)) was not called at all in VoiceAttack, and reading the related variables was also Not Set.

I tried to perform route planning to make a jump, and the obtained Log is as shown in the figure. Thank you for your help.

![圖片](https://github.com/user-attachments/assets/522b4960-6054-4698-9c83-6959e5ca5364)

**Reply by YenShihHung** [reply]
Is my understanding of this function wrong? When will this function be called?

https://youtu.be/W4E6agJ6ZFY

**Reply by Tkael** [reply, marked as answer]
The `Nav route` event (https://github.com/EDCD/EDDI/wiki/Nav-route-event) is invoked when you plot a route via the in-game menus.

The `Route details` event (https://github.com/EDCD/EDDI/wiki/Route-details-event) is triggered when you use EDDI to plot your route (for example via the `route` VoiceAttack plugin command (https://github.com/EDCD/EDDI/wiki/VoiceAttack-Integration#route).
